source/_components/ptvsd.markdown

```diff
@@ -0,0 +1,109 @@
+---
+layout: page
+title: "PTVSD Debugger"
+description: "Debugging from visual studio code."
+date: 2019-04-24 09:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: home-assistant.png
+ha_category: "Utility"
+redirect_from: /components/ptvsd/
+ha_qa_scale: internal
+ha_release: 0.92
+---
+
+The `ptvsd` component allows you to connect the ptvsd debugger from visual studio code to connect to home assistant.
+
+To enable the `ptvsd` component add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
```

Comment:
Maybe better if you place the other examples under the configuration variables section.